docs/Config_Reference.md

```diff
@@ -1128,7 +1128,43 @@ the nature of skew correction these lengths are set via gcode. See
 [skew_correction]
 ```
 
-## Customized homing
+## [frame_expansion_compensation]
+
+Compensate for vertical toolhead movement caused by thermal expansion of the
+frame in real-time using a temperature probe coupled to a vertical member.
+E.g. A glass bead thermistor suspended halfway down the central channel of an
+2020 aluminum extrusion.
+
+See also: [extended g-code commands](G-Codes.md#Frame-Expansion-Compensation).
+
+```
+[frame_expansion_compensation]
+#temp_coeff:
+#   The temperature coefficient of expansion, in mm/K. For example, a
+#   temp_coeff of 0.01 mm/K will move the Z axis downwards by 0.01 mm for every
+#   Kelvin/degree celcius that the frame temperature increases. Defaults to 0.0,
+#   no offset.
+#temp_sensor:
+#   Temperature sensor to use for frame temp measurement. Use full config
+#   section name without quoutes. E.g. temperature_sensor frame
+#smooth_time:
+#   Smoothing window applied to the temp_sensor, in seconds. Can reduce motor
+#   noise from excessive small corrections in response to sensor noise. The
+#   default is 2.0 seconds.
+#max_comp_z:
+#   Disables compensation above this Z height [mm]. The last computed correction
+#   will remain applied until the toolhead moves below the specified Z position
+#   again. The default is 0.0mm (always on).
+#max_z_offset:
+#   Maximum absolute compensation that can be applied to the Z axis [mm]. The
+#   default is 99999999.0mm (unlimited).
+z_stepper:
```

Comment:
What if there are multiple Z steppers? from this description I would not understand how to interpret this setting

Reply:
Indeed confusing, that's why this config option has been removed and is no longer required for the module internally (I'm just holding off on doc rework until after I'm happy with the code).